### PR TITLE
Trying to require obspy version

### PR DESCRIPTION
**Pull request by mhearne-usgs**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mhearne-usgs** [comment]
@ocefpaf I'm trying to get this package to depend on obspy version 1.1.1 or higher, but if I put that in the recipe I get all kinds of cascading conflicts in the Azure builds. Is there a way out of this?

**Comment by ocefpaf** [comment]
> @ocefpaf I'm trying to get this package to depend on obspy version 1.1.1 or higher, but if I put that in the recipe I get all kinds of cascading conflicts in the Azure builds. Is there a way out of this?

`obspy` is pinned to a really old `proj4`, see https://github.com/conda-forge/obspy-feedstock/blob/e05aac0c4cf3ce89fa98a0e3ac72e8f7e45ee4f6/recipe/meta.yaml#L52.

I will be quite difficult to rebuild that version of `proj4` with modern packages. It will probably be easier to fix `obspy` code to work with a modern `proj4`.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ source:
   sha256: fd474c195b89b0e47e18851a94c9fa45247b77c37dc4a6806e58e05dfa679f7a
 
 build:
-  number: 0
+  number: 2
```
Comment (ocefpaf) [inline recipe/meta.yaml]: No need to jump from `0` to `2`. The next build is when you merge, so this is still `1`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -25,7 +25,7 @@ requirements:
     - libcomcat
     - prov
     - scipy
-    - obspy
+    - obspy >=1.1.1
```
Comment (ocefpaf) [inline recipe/meta.yaml]: I pushed a new `proj4`, not sure if that will help. Fingers crossed.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by mhearne-usgs** [comment]
@ocefpaf Thanks, I wasn't sure what I needed to do to restart the PR

**Comment by ocefpaf** [comment]
> @ocefpaf Thanks, I wasn't sure what I needed to do to restart the PR

Most of the time just closing and re-opening will do the trick. But here I took the opportunity to fix the Python `matrix` and re-render. (See the last two commits.)

**Comment by mhearne-usgs** [comment]
So I see that proj4 is still a problem (cartopy wants a newer one?). With all of these errors, how can you tell which package(s) is *the* problem?

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=58438&view=logs

**Comment by ocefpaf** [comment]
> So I see that proj4 is still a problem (cartopy wants a newer one?). With all of these errors, how can you tell which package(s) is _the_ problem?

It is not easy to debug that. Sorry. However, I'm pretty sure we have cartopy for `proj4` 4 and 5, and 6 on Linux.

Looks like Python 3.6 is OK. I can debug the packages that must be rebuilt later but if you are using only Python 3.6 you can merge this as-is. Just open an issue to track the Pyton 3.7 failure.

**Comment by mhearne-usgs** [comment]
ok, will do. I submitted an issue on the obspy feedstock repo to see if they can fix that dependency.

**Pull request merged**